app/ide-desktop/lib/dashboard/src/authentication/src/config.ts

```diff
@@ -0,0 +1,63 @@
+/** @file Configuration definition for the Dashboard. */
+
+import * as auth from './authentication/config'
+
+// =================
+// === Constants ===
+// =================
+
+/** The current environment that we're running in. */
+export const ENVIRONMENT: Environment = 'production'
+
+/** All possible URLs used as the OAuth redirects when running the cloud app. */
+const CLOUD_REDIRECTS = {
+    development: 'http://localhost:8081' as auth.OAuthRedirect,
```

Comment:
This port is hard-coded because it's the port react uses when running the application with `npm run dev`. We don't use a random port in that scenario, and it's development-environment specific. The port can't be random in this case because Cognito expects us to define acceptable redirect ports ahead-of-time. That is, when the Cognito infrastructure is brought up. If the two don't match, the redirect URLs don't work.

@PabloBuchu @somebody1234 thoughts?

Reply:
Like @indiv0 said this one localhost refers to redirect url for **local development environment** only. @wdanilo it is not referring to the localhost the electron app is running, is that ok for you then?

Reply:
Add comment that it is not used by electron, explain the flow

Reply:
comment seems to be there now